.env.example

```diff
@@ -5,15 +5,18 @@ REYA_WS_URL=
 PRIVATE_KEY=
```

Comment:
will be great if @0xZenus and @ioanaroceanu2000 can find time to review this PR as well